labman/db/process.py

```diff
@@ -2861,7 +2872,8 @@ def _generate_shotgun_sample_sheet(self):
                 bcl2fastq_sample_ids, i7_names, i7_sequences, i5_names,
                 i5_sequences, sample_proj_values, wells=wells,
                 sample_plates=sample_plates, description=samples_contents,
-                lanes=[lane], sep=',', include_header=include_header))
+                lanes=[lane], sep=',', include_header=include_header,
+                include_lane=(self.lane_count > 1)))
```

Comment:
Currently, the "self.lane_count > 1" check occurs three different places, in two different functions. I am worried this leaves us open to future bugs due to someone only updating some of the occurrences.

I suggest you turn the lane_count int property into an include_lane boolean property (after all, the only time the lane_count is used is to determine whether to include the header.)  Then you can just reference that property rather than hardcoding the check into multiple locations.